tools/depends/target/dav1d/Makefile

```diff
@@ -1,6 +1,24 @@
-include ../../Makefile.include
+-include ../../Makefile.include
 include DAV1D-VERSION
-DEPS = ../../Makefile.include Makefile DAV1D-VERSION
+DEPS = Makefile DAV1D-VERSION
+
+ifeq ($(CROSS_COMPILING), yes)
+  DEPS += ../../Makefile.include
+else
+  ROOT_DIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+  PREFIX := $(ROOT_DIR)/dvdcss-install
+  BASE_URL := http://mirrors.kodi.tv/build-deps/sources
+  AUTORECONF := autoreconf
+
+  ifeq ($(PLATFORM),)
+    PLATFORM = native
+    TARBALLS_LOCATION = $(ROOT_DIR)
+  endif
+endif
+
+
+RETRIEVE_TOOL := curl -Ls --retry 10 --retry-delay 3 -o $(ARCHIVE)
+ARCHIVE_TOOL := tar --strip-components=1 -xf
```

Comment:
can you move these two inside the `else` block. They aren't needed when doing a depends build

Reply:
they are only set if not already defined.